lua/apisix/core/profile.lua

```diff
@@ -0,0 +1,34 @@
+--
+-- Licensed to the Apache Software Foundation (ASF) under one or more
+-- contributor license agreements.  See the NOTICE file distributed with
+-- this work for additional information regarding copyright ownership.
+-- The ASF licenses this file to You under the Apache License, Version 2.0
+-- (the "License"); you may not use this file except in compliance with
+-- the License.  You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+
+local _M = {
+    version = 0.1,
+    profile = os.getenv("APISIX_PROFILE"),
+    apisix_home = ngx and ngx.config.prefix()
```

Comment:
Do we need to set a default path if the `profile.lua` was not running in OpenResty ENV.

Reply:
Ok, I have been set the value `(ngx and ngx.config.prefix()) or ""` for the var apisix_home